phpBB/phpbb/storage/adapter/local.php

```diff
@@ -17,97 +17,127 @@
 
 class local implements adapter_interface
 {
-	protected $filesystem;
+	protected $symfony_filesystem;
 
+	/**
+	 * Constructor
+	 */
 	public function __construct()
 	{
-		$this->filesystem = new \phpbb\filesystem\filesystem();
+		$this->symfony_filesystem	= new \Symfony\Component\Filesystem\Filesystem();
```

Comment:
why not using phpbb filesystem? 
And `use Symfony\Component\Filesystem\Filesystem;`